src/main/java/com/netflix/imflibrary/st2067_2/ApplicationTsp2121Composition.java

```diff
@@ -0,0 +1,268 @@
+/*
```

Comment:
Can you avoid adding TSP 2121 code in the 2067-2 folder? Can you use a separate folder? Also can you clarify why this is not in the other PR https://github.com/Netflix/photon/pull/263 ?

Reply:
Right for a new folder !
This PR is only for the HLG support in App2e, but it requires updates from PR #263 that's why it's a new branch from the TSP 2121, and it includes all commits. If we merge the TSP 2121 first, this PR will contains only the latest commit.

Reply:
And what about the App5 stored also in the App2 folder ?